sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,35 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId, baseURL) {
```

Comment:
```suggestion
  async getGlobalConf(gatewayId) {
```

You don't need to pass the `baseURL`. You can get this information from the `stackConfig` that is passed to the SDK during initialization.